tests/test_integration.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 D-Wave Systems Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import subprocess
+import unittest
+import os
+
+class IntegrationTests(unittest.TestCase):
+
+    def test_antenna_selection(self):
+        output = subprocess.check_output(["python", "antennas.py"])
```

Comment:
This is test is also an improvement to the existing `test_antennas.py` code. Could you simply make your changes in `test_antennas.py` as both files simply do a smoke test (yours is a more in-depth version!).

Reply:
@m3ller my plan is to add more integration tests , so created separate file